internal/providersdkv2/data_source_hvn_route.go

```diff
@@ -58,6 +58,26 @@ Remove this attribute from the configuration for any affected resources.
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
+			"azure_config": {
+				Description: "The azure configuration for routing.",
+				Type:        schema.TypeList,
+				Computed:    true,
+				Optional:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"next_hop_type": {
+							Description: "The type of Azure hop the packet should be sent to.",
+							Type:        schema.TypeString,
+							Computed:    true,
```

Comment:
It's been a while since I've done provider work but I thought computed referenced things that were not user-configurable? Is that the case here?